### PR TITLE
Enable skipped tests

### PR DESCRIPTION
**Pull request by iSazonov**
# PR Summary

- Replace `$PSDefaultParameterValues` with `$global:PSDefaultParameterValues` to correctly restore skipping settings.
- Skip FileSystem Provider Extended tests on Linux because Hidden flag is not supported on Linux in .Net 5 Preview. The tests should be fixed in follow PR.
- Pending two Linux tests because of .Net 5.0 Preview bug dotnet/runtime#33673.
(PowerShell process name is `ConsoleHost mai` instead of `pwsh`)
- (Tracking issue #12908) Pending one Linux test because of PowerShell bug manifested due to mentioned .Net bug
https://github.com/PowerShell/PowerShell/blob/5ef69e88bc2a6e500349a8eec8d13bb27e143e3e/src/System.Management.Automation/CoreCLR/CorePsPlatform.cs#L972-L985
Since a process names contains a space ( `ConsoleHost mai`) the split returns additional extra element and we get wrong value with `parts[3]`.

## PR Context

I discovered that CIs skip silently many tests which run after the Clipboard tests. 

I have more general concern - I saw that tests was skipped on Linux but not on Windows and MacOS and I don't understand how this can be. My concern is that there is a bug. I hope MSFT can investigate this.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1 [hunk 1]
```diff
@@ -13,7 +13,7 @@ Describe 'Clipboard cmdlet tests' -Tag CI {
         }
```
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:13]: @iSazonov, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://docs.universaldashboard.io should work
```powershell
Name or service not known
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:13]: @iSazonov, your last commit had 52 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path
```powershell
Expected 4, but got 5.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 46
46:             $result.Count | Should -Be 4
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path -Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 52
52:             $result.Count | Should -Be 1
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path -Attribute Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 59
59:             $result.Count | Should -Be 1
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Directory/-File.Get-ChildItem -Path -File
```powershell
Expected 2, but got 3.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 79
79:             $result.Count | Should -Be 2
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Directory/-File.Get-ChildItem -Path -File -Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 84
84:             $result.Count | Should -Be 1
```

Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:13]: @iSazonov, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://docs.universaldashboard.io should work
```powershell
Name or service not known
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:13]: @iSazonov, your last commit had 3 failures in `PowerShell-CI-linux`
Get-Process.Test for process property = Name
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   15
Strings differ at index 0.
Expected: 'pwsh'
But was:  'ConsoleHost mai'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 74
74:         (Get-Process -Id $PID).Name | Should -BeExactly "pwsh"
```
Process Parent property.Has Parent process property
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 128
128:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent' | Should -Not -BeNullOrEmpty
```
Process Parent property.Has valid parent process ID property
```powershell
Expected 7727, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 133
133:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent.Id' | Should -Be $PID
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
Please investigate test failures

**Comment by iSazonov** [comment]
@TravisEz13 two tests fail in Get-Process.Tests.ps1 - I do not find when we ruined it and how to fix it.

File: test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1 (hunk 1 above)
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:13]: @iSazonov, your last commit had 3 failures in `PowerShell-CI-linux`
Get-Process.Test for process property = Name
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   15
Strings differ at index 0.
Expected: 'pwsh'
But was:  'ConsoleHost mai'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 74
74:         (Get-Process -Id $PID).Name | Should -BeExactly "pwsh"
```
Process Parent property.Has Parent process property
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 128
128:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent' | Should -Not -BeNullOrEmpty
```
Process Parent property.Has valid parent process ID property
```powershell
Expected 7780, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 133
133:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent.Id' | Should -Be $PID
```

**Comment by TravisEz13** [comment]
@PoshChan please rerun windows

**Comment by PoshChan** [comment]
@TravisEz13, successfully started rebuild of `PowerShell-CI-Windows`

**Comment by iSazonov** [comment]
@TravisEz13 @SteveL-MSFT I found why Linux tests fail. It is https://github.com/dotnet/runtime/issues/33673. 
I hope we get the fix in next .Net 5 Preview. While I put the tests in pending state.

**Comment by TravisEz13** [comment]
Please file an issue to track re-enabling the tests.

**Comment by therajdeepbiswas** [comment]
I have found a solution to #12908. I would like to contribute.

**Pull request merged**